docs/intro.rst

```diff
@@ -116,13 +118,38 @@ If the same property is set with multiple methods,
 the order of precedence is command-line properties, external properties file and
 classpath properties file. You can find a list of settings in :ref:`configuration section<camus_config>` .
 
+The recommended deployment method is to use the ``camus-run`` script to initiate the MapReduce job:
+
+.. sourcecode:: bash
+
+   $ bin/camus-run -D <property=value> \
+      -P <path to external properties file> \
+      -p <path to properties file from classpath>
+
+If you need more control you can run the job yourself, but will have to configure some parameters
+and environment variables yourself (or reuse the ``bin/camus-config`` script to generate the
```

Comment:
bin/camus-config will not set the environment variables outside the script. Also, it would be nice to describe more about the what the script is doing.